src/administration/Administration.Service/BusinessLogic/ConnectorsBusinessLogic.cs

```diff
@@ -433,6 +435,19 @@ private async Task UpdateConnectorUrlInternal(Guid connectorId, ConnectorUpdateR
             con.ConnectorUrl = data.ConnectorUrl;
         });
 
+        if (connector.SelfDescriptionDocumentId != Guid.Empty)
+        {
+            documentRepository.AttachAndModifyDocument((Guid)connector.SelfDescriptionDocumentId!, null, doc =>
```

Comment:
Please change to .Value for easier readability

```suggestion
            documentRepository.AttachAndModifyDocument(connector.SelfDescriptionDocumentId.Value, null, doc =>
```

Reply:
Changed